OsmAnd/res/values/strings.xml

```diff
@@ -13,38 +13,38 @@
 -->
 
     <string name="shared_string_mini">mini</string>
-    <string name="sample_wikivoyage">Sample Wikivoyage</string>
+    <string name="sample_wikivoyage">Wikivoyage example</string>
     <string name="travel_route_types">Route types</string>
     <string name="shared_string_files">Files</string>
-    <string name="service_is_not_available_please_try_later">Service is not available, please try later</string>
-    <string name="mapillary_menu_filter_description_new">Filter images by date or by type. Only active in closeup zoom.</string>
```

Comment:
This should have been "(Only active in)" if done this way, but it isn't that good.